VDA-XP-Reward/.env.example

```diff
@@ -0,0 +1,17 @@
+# Used to verify contract after deployment
+ETHERSCAN_API_KEY="<Input your etherscan.com api key>"
+POLYGONSCAN_API_KEY="<Input your polygonscan.com api key"
+
+# Used to deploy contract
+PRIVATE_KEY="<Input your wallet private key for deploying contract>"
```

Comment:
Is this a Verida private key?

Reply:
No, this is the wallet private key for paying gas fees while deploying the contract.

Reply:
Ok, make it clear this is a polygon private key